sdk/state/state.go

```diff
@@ -31,8 +31,15 @@ type Channel struct {
 	observationPeriodTime      time.Duration
 	observationPeriodLedgerGap int64
 
-	startingSequence int64
-	// TODO: balances         []Amount
+	startingSequence        int64
+	iterationNumber         int64
+	iterationNumberExecuted int64
+	// TODO - is this the best way to rep Balance? (perspective of initiator/responder and not Me/Other)
+	// The balance owing from the initiator to the responder, if positive, or
+	// the balance owing from the responder to the initiator, if negative.
+	// TODO - use Balance struct
+	Balance int64
+	Asset   Asset
```

Comment:
You can use an `Amount` type here, it captures both the asset and amount value in the struct. That'll make it easier for us to add support for multiple balances.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/55/commits/9add63d410c809c7cda00cedecd510ea5a83b87f

proposal to name the struct `Balance` instead of `Amount`? `channel.amount.Amount` feels weird, and a "balance" per asset makes sense